src/components/command-bar/commands/search/unified-search/components/unified-hit/unified-hit.module.css

```
@@ -0,0 +1,9 @@
+.withHighlight {
+	color: black;
```

Total nit: but this is probably the value we want to make it work with theming. Totally cool if you want to adjust when actually doing the UI later on 👌🏻 
```suggestion
	color: var(--token-color-foreground-primary);
```

---

I dig it - made some updates so the results card work in dark mode 👍